test/MonitoringTest.py

```diff
@@ -0,0 +1,14 @@
+"""Test that monitoring tools are working"""
+from MediawikiTest import MediawikiBase
+
+
+class GrafanaTest(MediawikiBase):
+    def test1(self):
+        """Test that Grafana is running."""
+        status = self.getUrlStatusCode("http://grafana:3000")
```

Comment:
obtain ports for test from env variable

Reply:
But these are the internal ports inside the docker network, while the purpose of the ports defined in the env file is to map the internal ports to a port on localhost. or not?